src/app/game/tic-tac-toe/multi-player/page.js

```
@@ -208,6 +289,18 @@ const Game = () => {
                 ))}
             </div>
 
+            {(winner || isBoardFull) && (
+  <div style={{ marginTop: 20 }}>
+    <button
+      className={styles.copyButton}
+      onClick={() => reset(true)}
+      disabled={loading || waitingForOpponent}
+    >
+      {waitingForOpponent ? 'Waiting for opponent...' : 'Play Again'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)